### ISSUE TITLE
NVDA does not read in the Fast Log Entry

### ISSUE THREAD

**Issue by HegrJan** [nvaccess/nvda#8996]
### Steps to reproduce:
-1. Download Fast Log Entry installer from http://df3cb.com/fle/
0. Install FLE
1. Open FLE
2. Try to navigate using cursor keys

### Actual behavior:
The text (log content) is not read. There are unknown symbols on the braille display eg. '\x2023''\x6548''\x6461''\x7265'...

### Expected behavior:
An edit field similar to Notepad filled with example log should be readable.

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
2018.3.2

#### Windows version:
Windows 7 64bit Version 6.1.7601 Service Pack 1 Build 7601

#### Name and version of other software in use when reproducing the issue:
Fast Log Entry 2.12

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
- NVDA 2017.3, Windows XP 32bit - works
- NVDA 2018.3.2, Windows 10 32bit - doesn't work
- NVDA 2018.3.2, Windows 10 64bit - doesn't work

### Remarks
- Commercial screenreaders work properly.

**Comment by HegrJan** [comment]
Problem still persists in NVDA 2019.1

**Comment by HegrJan** [comment]
Problem still persists in NVDA 2019.3.1

**Comment by Adriani90** [comment]
This seems to be a regression.
cc: @jcsteh, @michaelDCurran this regression is quite a long time ago, but maybe someone of you can find the coresponding cause for this.

**Comment by Adriani90** [comment]
@HegrJan I assume this is still reproducible. Is this correct?

**Comment by HegrJan** [comment]
Yes, it is.

**Comment by HegrJan** [comment]
Problem still persists in NVDA 2020.1

**Comment by lukaszgo1** [comment]
@HegrJan Just to make sure this is fixed for you coulld you please test with this [try build](https://ci.appveyor.com/api/buildjobs/4ym0fbmj2tubts35/artifacts/output%2Fnvda_snapshot_pr11488-20731%2C594e2dab.exe)?

**Comment by HegrJan** [comment]
Yes, it works like a charm (Windows 7 64bit). Thank you!
